assets/app/views/directives/header/default-header.html

```diff
@@ -0,0 +1,17 @@
+<nav class="navbar navbar-pf-alt" role="navigation">
+  <div class="navbar-header">
+    <!-- mobile nav menu -->
+    <div row flex class="navbar-flex-btn toggle-menu">
+      <button type="button" class="navbar-toggle project-action-btn ng-isolate-scope" data-toggle="collapse" data-target=".navbar-collapse-2">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+    </div>
+    <a class="navbar-brand" id="openshift-logo" href="./">
+      <div id="header-logo"></div>
+    </a>
+  </div>
```

Comment:
@spadgett 
In reference to https://bugzilla.redhat.com/show_bug.cgi?id=1301478 the id hasn't changed, though the markup is no longer in index.html. Not sure if that makes a difference.